src/main/java/com/mixpanel/android/viewcrawler/GestureTracker.java

```diff
@@ -0,0 +1,91 @@
+package com.mixpanel.android.viewcrawler;
+
+import android.app.Activity;
+import android.view.MotionEvent;
+import android.view.View;
+import com.mixpanel.android.mpmetrics.MixpanelAPI;
+
+/**
+ * Tracking ABTesting Gestures
+ */
+public class GestureTracker {
+	public GestureTracker(MixpanelAPI mMixpanel, Activity parent) {
+		trackGestures(mMixpanel, parent);
+	}
+
+	private void trackGestures(final MixpanelAPI mMixpanel, Activity parent) {
+		parent.getWindow().getDecorView().findViewById(android.R.id.content).setOnTouchListener(new View.OnTouchListener() {
+
+			private void resetGesture() {
+				mFirstToSecondFingerDifference = -1;
+				mSecondFingerTimeDown = System.currentTimeMillis();
```

Comment:
Then I don't see the point of initializing a variable to the current time in ms. Let's change it :)